specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2022-06-01-preview/machineLearningServices.json

```diff
@@ -2157,10 +2157,11 @@
             "Deleting",
             "Succeeded",
             "Failed",
-            "Canceled"
+            "Canceled",
+            "SoftDeleted"
           ],
           "x-ms-enum": {
-            "name": "ProvisioningState",
+            "name": "WorkspaceProvisioningState",
```

Comment:
>WorkspaceProvisioningState [](http://example.com/codeflow?start=21&length=26)

ARM does not recognize other names of this property. retain this as provisioningstate

Reply:
This is the name of this enumeration type and it only affects the generated SDK. The property of this thing still remains. Therefore I believe this change will not affect ARM